apps/nova/src/components/Tutors/Content.tsx

```diff
@@ -33,9 +34,22 @@ const Content: React.FC<{
   const openBookingDialog = useCallback((tutor: Tutor) => setTutor(tutor), []);
   const closeBookingDialog = useCallback(() => setTutor(null), []);
 
-  if (loading) return <Loading className="h-[30vh]" />;
-  // todo: add error component
-  if (error) return "ERROR: TODO";
+  if (loading)
+    return (
+      <div className="mt-[15vh]">
+        <Loader size="large" text={intl("tutors.loading")} />;
```

Comment:
The changes should respect the mobile layout as well. On mobile, the size should `small` (the same thing applies to the error case)